data/api.py

```diff
@@ -149,23 +149,10 @@ class DDSEndpointViewSet(viewsets.ModelViewSet):
     queryset = DDSEndpoint.objects.all()
 
 
-class DDSUserCredViewSet(viewsets.ModelViewSet):
+class DDSUserCredViewSet(viewsets.ReadOnlyModelViewSet):
     permission_classes = (permissions.IsAuthenticated,)
-    serializer_class = DDSUserCredSerializer
-    filter_backends = (DjangoFilterBackend,)
-    filter_fields = ('user',)
-
-    def get_queryset(self):
-        return DDSUserCredential.objects.filter(user=self.request.user)
-
-    def perform_create(self, serializer):
-        self.save_with_user(serializer)
-
-    def perform_update(self, serializer):
-        self.save_with_user(serializer)
-
-    def save_with_user(self, serializer):
-        serializer.save(user=self.request.user)
+    serializer_class = ReadOnlyDDSUserCredSerializer
+    queryset = DDSUserCredential.objects.all()
```

Comment:
The only credentials stored here will be for the bespin user created by an admin user.
They can't see the token just see the name and the id.
Initially there will only be a single credential value for production DukeDS.
Long term they may want to pick from different DukeDS instances.